File: Python/File program/File writing program.py
try:
	filedriver = open("/storage/emulated/0/Text.txt","w")
	text = ""
	while True:
		 text = input()
		 if text == "save" or text == "Save":
		 	break
		 filedriver.write(text+"\n")
	print("----------")
	print("File saved")
	print("----------")
except:
		 print("File creation error!")
finally:
		 filedriver.close()

File: c++/fileProgram/FileWritingProgram .cxx
#include <iostream>
#include <fstream>
using namespace std;
int main()
{
	ofstream filedriver("/storage/emulated/0/New file.txt",ios::out);
	string file = "my name is piyal.\n";
	filedriver<<file;
	filedriver.close();
	return 0;
}